data/fields/fuel/fuel_multi.json

```diff
@@ -0,0 +1,66 @@
+{
+    "key": "fuel:",
+    "type": "multiCombo",
+    "label": "Fuel Types",
+    "strings": {
+        "options": {
+          "diesel": "Diesel",
+          "gtl_diesel": "GTL Diesel",
+          "hgv_diesel": "HGV Diesel",
+          "biodiesel": "Bio Diesel",
+          "diesel:class2": "Cold Weather Diesel",
+          "taxfree_diesel": "Untaxed Diesel",
+          "octane_80": "Octane 80",
```

Comment:
A bunch of these values need to be tweaked for American English. For starters, it would be “80 Octane”, not “Octane 80”. I think `octane_` is merely being used as a namespace.

Reply:
Also it probably makes sense to add strings for all the values between 80 and 100, if not beyond, because the specific octane numbers sold vary by country, region, or city, or brand.